src/platform/packages/shared/kbn-server-route-repository/src/register_routes.ts

```diff
@@ -156,6 +154,10 @@ export function registerRoutes<TDependencies extends Record<string, any>>({
           opts.body.attributes.data = error?.data;
         }
 
+        if (opts.statusCode >= 500) {
```

Comment:
I'm not sure if it's worth to log the other errors under debug

Reply:
Are all status codes other than 500 going to be logged by core in this case?